### PR TITLE
Redirect 'Email-ext Recipes' page from wiki to GitHub docs #3737

### PR DESCRIPTION
**Pull request by Fargk**
See [issue #3737](https://github.com/jenkins-infra/jenkins.io/issues/3737)

### CODE REVIEW

File: dist/profile/templates/confluence/vhost.conf [hunk 1]
```diff
@@ -1572,6 +1572,7 @@ RewriteRule "^/display/JENKINS/Installing\+Jenkins\+with\+Docker$" "https://www.
 RewriteRule "^/display/JENKINS/Plugins$" "https://plugins.jenkins.io/" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Release\+Process$" "https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#release-process" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Email-ext+Recipes$" "https://github.com/jenkinsci/email-ext-plugin/blob/master/docs/recipes.adoc" [NE,NC,L,QSA,R=301]
```
Comment (MarkEWaite) [inline dist/profile/templates/confluence/vhost.conf]: Need to escape the 'plus' character in the pattern match.
```suggestion
RewriteRule "^/display/JENKINS/Email-ext\+Recipes$" "https://github.com/jenkinsci/email-ext-plugin/blob/master/docs/recipes.adoc" [NE,NC,L,QSA,R=301]
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Need to escape the `+` character in the source URL

File: dist/profile/templates/confluence/vhost.conf (hunk 1 above)
Comment (Fargk) [reply dist/profile/templates/confluence/vhost.conf]: oh, thanks!

Comment (Fargk) [reply dist/profile/templates/confluence/vhost.conf]: Fixed

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**